src/relay/op/nn/convolution.cc

```diff
@@ -1142,5 +1142,51 @@ TVM_REGISTER_GLOBAL("relay.op.nn._make.deformable_conv2d")
 .set_body_typed(MakeDeformableConv2D);
 
 
+bool Conv2DVNNIRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
```

Comment:
Can you call Conv2DRel itself?

Reply:
Since this op requires packing, `NHWCwc` conv `HWOCoc` = `NHWO`. It is kinda trick to use existing infra.